web-api/PetProjectWepApi/PetProjectWepApi/Infrastructure/Logging/logs/file-20221116.txt

```diff
@@ -0,0 +1,30 @@
+Application startup exception 11/16/2022 5:22:08 PM
```

Comment:
You should not add the log files to the project, please add the folder to git ignore.